chain/index/events.go

```diff
@@ -563,6 +566,9 @@ func makePrefillFilterQuery(f *EventFilter) ([]any, string, error) {
 				clauses = append(clauses, "("+strings.Join(subclauses, " OR ")+")")
 			}
 		}
+	} else {
+		clauses = append(clauses, "ee.codec=?")
+		values = append(values, uint64(multicodec.Raw))
```

Comment:
This won't work because we call this method from both the eth APIs and also `GetActorEventsRaw` and `SubscribeActorEventsRaw` and these APIs want any codec if they don't specify a `KeysWithCodec`.

I think what you might need to do is extend `EventFilter` to add a `OnlyRaw`, or `WithCodec` or something like that so that when this gets called from the Eth APIs, that can be set so we can restrict to Raw only.

Reply:
yeah, for now just `Codec` is fine, "only used if KeysWithCodec is not set"